index.html

```diff
@@ -11,86 +11,139 @@
 </head>
 
 <body>
-
-	<h1>You got this!  Good luck.</h1>
-
-<!-- Menu Content 
-S&J
-
-Home
-Services
-Contact
--->
-
-
-<!-- Home Page Content:
+<div class="container"> <!-- Container -->
+	<!-- Menu Content -->
```

Comment:
For page specific styling, add a page class to the outer container that you can target in your LESS files (i.e. `class= container home` here)